typescript.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   "extends": [
-    "prettier/@typescript-eslint"
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
```

Comment:
sine we want this for every project, I think this needs to go to `index.js` instead of `typescript.js`

Reply:
Agreed!

Reply:
Updated